backend/apps/client/src/submission/submission.service.ts

```diff
@@ -54,13 +54,13 @@ export class SubmissionService implements OnModuleInit {
             Array.isArray(error) &&
             error.every((e) => e instanceof ValidationError)
           ) {
-            this.logger.error('Message format error', { ...error })
+            this.logger.error(error, 'Message format error')
             return new Nack()
           } else if (error instanceof UnprocessableDataException) {
-            this.logger.error('Iris exception', error)
+            this.logger.error(error, 'Iris exception')
             return new Nack()
           } else {
-            this.logger.error('Unexpected error', error)
+            this.logger.error(error, 'Unexpected error')
             return new Nack()
           }
```

Comment:
이 PR의 문제는 아니긴 한데, `return new Nack()` if문 바깥으로 빼주세요!
```suggestion
          ) {
            this.logger.error(error, 'Message format error')
          } else if (error instanceof UnprocessableDataException) {
            this.logger.error(error, 'Iris exception')
          } else {
            this.logger.error(error, 'Unexpected error')
          }
          return new Nack()
```